packages/salesforcedx-vscode-core/test/vscode-integration/commands/templates/forceApexClassCreate.test.ts

```diff
@@ -18,7 +18,7 @@ import { forceApexClassCreate } from '../../../../src/commands/templates/forceAp
 import { nls } from '../../../../src/messages';
 import { notificationService } from '../../../../src/notifications';
 import { telemetryService } from '../../../../src/telemetry';
-import { getRootWorkspacePath } from '../../../../src/util';
+import { ConfigUtil, getRootWorkspacePath } from '../../../../src/util';
```

Comment:
No, telemetry test is only for apex class create since the other template types are very similar. It might be belonging to its own test file though, but not a pressing need to refactor now.